.well-known/security.txt

```diff
@@ -0,0 +1,12 @@
+# Our security address
+Contact: mailto:erlang-security@erlang.org
+Contact: https://github.com/erlang/otp/security
+
+Canonical: https://www.erlang.org/.well-known/security.txt
+
+Preferred-Languages: en
+
+# Our security policy
+Policy: https://github.com/erlang/otp/blob/master/SECURITY.md
+
+Expires: 2025-04-01T00:00:00z
```

Comment:
requires a manual update every year.
supposed to be done manually or automatically on some frequent occasion.

pls update date to October/Nov?